nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/provider/flow/git/GitFlowMetaData.java

```diff
@@ -176,6 +190,13 @@ public void remoteRepoExists(String remoteRepository) throws IOException {
         }
     }
 
+    /**
+     * If validation of remote clone repository throws no exception then clone the given 'Remote Clone Repository' in
```

Comment:
Better:

Clone the repository given in the 'Remote Clone Repository' configuration.